scaling/plugins/sge.py

```diff
@@ -0,0 +1,46 @@
+import subprocess
+import shlex
+import logging
+import json
+
+log = logging.getLogger(__name__)
+
+# get nodes requested from pending jobs
+def pending(instance_type):
+    command = "qstat -g d -s p -u '*'"
+    slots = run_command(command)
+    return nodes(slots, instance_type)
+
+# get nodes requested from pending jobs
```

Comment:
The comment needs to be changed (the same for the `running()` functions of the other modules).

Reply:
changed